localstack/services/apigateway/apigateway_listener.py

```diff
@@ -273,19 +157,6 @@ def forward_request(self, method, path, data, headers):
 
         return True
 
-    def return_response(self, method, path, data, headers, response):
-        try:
-            response_data = json.loads(response.content)
-            # Fix an upstream issue in Moto API Gateway, where it returns `createdDate` as a string
```

Comment:
In this PR https://github.com/spulec/moto/pull/1457 `createdDate` was fixed and no longer string (I checked `moto-ext` has the latest version of `moto` so I guess we're good otherwise I can build the image locally and create an api to double check)